visivo/models/base/parent_model.py

```diff
@@ -146,9 +149,12 @@ def name_match(trace):
 
     @staticmethod
     def show_dag(dag):
```

Comment:
Huh, I never new we had a way to show the built dag

Reply:
I use it all the time!